core-bundle/src/Resources/contao/library/Contao/Search.php

```diff
@@ -55,7 +55,7 @@ public static function indexPage($arrData)
 		$arrSet['url'] = $arrData['url'];
 		$arrSet['title'] = $arrData['title'];
 		$arrSet['protected'] = $arrData['protected'];
-		$arrSet['filesize'] = $arrData['filesize'];
+		$arrSet['filesize'] = $arrData['filesize'] ?? 0;
```

Comment:
Should probably default to `null` for BC:
```suggestion
		$arrSet['filesize'] = $arrData['filesize'] ?? null;
```